admin/sql/CreateIndexes.sql

```diff
@@ -172,6 +172,8 @@ CREATE INDEX event_tag_idx_tag ON event_tag (tag);
 CREATE INDEX event_tag_raw_idx_tag ON event_tag_raw (tag);
 CREATE INDEX event_tag_raw_idx_editor ON event_tag_raw (editor);
 
+CREATE INDEX first_recording_release_date_idx ON first_recording_release_date (year, month, day);
```

Comment:
This index might be useful if we want to browse/sort recordings by year/date, but should probably be removed unless or until we have a plan for that. (That may only be useful in the context of artists, and this table doesn't have artist info.)

Reply:
Allowing to sort by date (or filter?) doesn't seem like a terrible idea, at least.